api/src/main/java/org/datacleaner/metadata/ColumnMeaning.java

```diff
@@ -63,8 +63,8 @@
 
     COMPANY_REGISTRATION_NUMBER("Company registration number", "Reg number"),
 
-    COMPANY_NAME("Company name", "Company", "Organization", "Business", "Organisation", "Employer", "Firm",
-            "Workplace", "Works at", "Vendor", "Supplier", "Customer", "Customer name"),
+    COMPANY_NAME("Company name", "Company", "Organization", "Business", "Organisation name", "Employer", "Firm",
+            "Workplace", "Works at", "Vendor name", "Supplier name", "Customer name"),
```

Comment:
Now that we don't tokenize column meanings isn't it a problem that _Supplier_ and _Vendor_ are only there in combination with name?